.github/workflows/angular.yml

```diff
@@ -75,70 +75,70 @@ jobs:
           - ngx-gradle-mongodb-kafka-cucumber
           - ngx-gradle-h2disk-ws-nocache
         include:
-          - app-type: ngx-default
+          - app-sample: ngx-default
             entity: sqlfull
             environment: prod
             war: 0
             e2e: 1
             testcontainers: 1
-            sonar-analyse: 1
```

Comment:
@mshima Did you remove sonar-analyse = '1' by mistake?

Reply:
No switch to `sonar-analyse: 0` at the additional bellow.